test/e2e/page-objects/pages/dialog/snap-install.ts

```diff
@@ -76,12 +90,29 @@ class SnapInstall {
       },
       { timeout: veryLargeDelayMs, interval: 100 },
     );
-    await this.driver.clickElement(this.nextPageButton);
+    await this.driver.clickElement(this.confirmButton);
+  }
+
+  async clickConnectButton() {
+    console.log('Clicking the connect button');
+    await this.driver.clickElement(this.connectButton);
+  }
+
+  async clickOkButton() {
+    console.log('Clicking Confirm/Ok button and wait for dialog to close');
+    await this.driver.clickElementAndWaitForWindowToClose(this.okButton);
```

Comment:
this fixes a race condition and mitigates flakiness, this affected lots of snap specs